server/server/models/subscribe.py

```diff
@@ -17,3 +17,27 @@ class Subscribe(Base):
 
     users = relationship("User", foreign_keys=(user_id,))
     events = relationship("Event", foreign_keys=(event_id,))
+
+    @classmethod
+    def get_subscription(cls, request, **kwargs):
+        subscription = request.dbsession.query(cls).filter_by(**kwargs).\
+                       one_or_none()
+        return subscription
```

Comment:
You don't need to create new variable, just `return request.dbsession.query(cls).filter_by(**kwargs).one_or_none()`